src/OrbitGl/ThreadTrack.cpp

```diff
@@ -139,7 +138,7 @@ std::string ThreadTrack::GetBoxTooltip(const Batcher& batcher, PickingId id) con
   std::string module_name = orbit_client_data::kUnknownFunctionOrModuleName;
   std::string function_name = orbit_client_data::kUnknownFunctionOrModuleName;
   if (func != nullptr) {
-    module_name = orbit_client_data::function_utils::GetLoadedModuleNameByPath(func->file_path());
+    module_name = orbit_client_data::ModuleData::GetLoadedModuleNameByPath(func->file_path());
```

Comment:
As per my other comments, this might change, but better here would be: `func->GetLoadedModuleName()`

Reply:
Done.